extension/oidcextension/README.md

```diff
@@ -0,0 +1,36 @@
+# OIDC
+
+This extension enables bearer token authorization for GRPC receivers.
```

Comment:
Not 100% happy with the module name. I think "oidcauthextension" would be my first option, but "authoidcextension" would keep all auth extensions together. What do you think?

Reply:
I like `authoidcexetention` to group things.